src/ontology/Makefile

```diff
@@ -1,7 +1,7 @@
 # ----------------------------------------
 # Makefile for mondo-ingest
 # Generated using ontology-development-kit
-# ODK Version: v1.5
+# ODK Version: v1.5-dev
```

Comment:
please use ODK_TAG=v1.4.3 as described on your ICD11 PR when running update_repo. There is too much spurious change in this PR.

Reply:
> Lets either merge https://github.com/monarch-initiative/mondo-ingest/pull/432 first

Yes, I put in the OP that we should merge #432 first.

> please use ODK_TAG=v1.4.3
Yes, my bad. I'm having several issues regarding that, and the appropriate area to discuss that is here:
- https://github.com/monarch-initiative/mondo-ingest/pull/432#discussion_r1482008978